src/software/ai/hl/stp/tactic/shadow_enemy/shadow_enemy_fsm.cpp

```diff
@@ -38,15 +39,37 @@ Point ShadowEnemyFSM::findBlockShotPoint(const Robot &robot, const Field &field,
 bool ShadowEnemyFSM::enemyThreatHasBall(const Update &event)
 {
     std::optional<EnemyThreat> enemy_threat_opt = event.control_params.enemy_threat;
+
     if (enemy_threat_opt.has_value())
     {
-        return enemy_threat_opt.value().has_ball;
-    };
-    LOG(WARNING) << "Enemy threat not initialized for robot " << event.common.robot.id()
-                 << "\n";
+        const double ENEMY_NEAR_BALL_DIST = 0.22;
```

Comment:
you can define this constant in the header, also can you explain a bit why/how this number was picked?